packages/public-api/src/packageMetadata/packageMetadataService.ts

```diff
@@ -1,23 +1,32 @@
 import { getDatabaseConnection, PackageMetadata } from '@gradejs-public/shared';
 
-export async function getPackageMetadataByPackageNames(packageNames: string[]) {
+type Metadata = Omit<Omit<PackageMetadata, 'fullDescription'>, 'versionSpecificValues'>;
+
+export async function getPackageMetadataByPackageNames(
+  packageNames: string[]
+): Promise<Record<string, Metadata>> {
   if (!packageNames.length) {
     return {};
   }
 
   const db = await getDatabaseConnection();
   const packageMetadataRepo = db.getRepository(PackageMetadata);
+  const columnsToSelect = packageMetadataRepo.metadata.columns
+    .map((col) => [col.databaseName, col.propertyName] as [string, keyof Metadata])
+    .filter((col) => !['full_description', 'version_specific_values'].includes(col[0]));
 
   const packageMetadataEntities = await packageMetadataRepo
     .createQueryBuilder('pm')
+    .select(columnsToSelect.map((col) => col[0]))
     .where('pm.name IN (:...packageNames)', { packageNames })
-    .getMany();
-
-  const packageMetadataMap = packageMetadataEntities.reduce((acc, val) => {
-    acc[val.name] = val;
+    .getRawMany();
 
+  return packageMetadataEntities.reduce((acc, val) => {
```

Comment:
```js
 const packageMetadataMap = packageMetadataEntities.reduce((acc, val) => {
    // comment why
    delete val.fullDescription;
    delete val.versionSpecificValues;
    acc[val.name] = val;
}
```

Reply:
Used the `select: false` solution eventually.